### ISSUE TITLE
broken handler for facebook event comment

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy#337]
specifically, https://brid-gy.appspot.com/comment/facebook/11500459/1566004873636047/1566004873636047_1566006050302596 currently 400s. thanks to @aaronpk for the report!

it's for this event comment: https://www.facebook.com/events/1566004873636047/permalink/1566004876969380/?comment_id=1566006050302596&offset=0&total_comments=4

we try to fetch object id `1566004873636047_1566006050302596` from the FB graph API, which FB naturally chokes on.

log:
```
Fetching facebook:11500459 comment ('1566004873636047', '1566004873636047_1566006050302596')
Fetching https://graph.facebook.com/1566004873636047_1566006050302596?access_token=CAAD..., kwargs {}
Error 400, response body: Bad Request
```

**Comment by snarfed** [comment]
event comments were implemented in #49

**Comment by snarfed** [comment]
[fetching the comment id alone works in the API explorer](https://developers.facebook.com/tools/explorer?method=GET&path=1566006050302596&version=v2.2&), ie `/1566006050302596`. let's do that.
